<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFsdSetupForm</class>
 <widget class="QFrame" name="CFsdSetupForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>82</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FSD setup</string>
  </property>
  <layout class="QGridLayout" name="gl_FsdSetupForm">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="4" column="1">
    <widget class="QLabel" name="lbl_AircraftParts">
     <property name="toolTip">
      <string>aircraft parts</string>
     </property>
     <property name="text">
      <string>Parts</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QCheckBox" name="cb_FastPositionSend">
     <property name="text">
      <string>send</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QCheckBox" name="cb_GndFlagSend">
     <property name="text">
      <string>send</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="lbl_GndFlag">
     <property name="text">
      <string>Gnd.flag</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="cb_AircraftPartsReceive">
     <property name="text">
      <string>receive</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QCheckBox" name="cb_FastPositionReceive">
     <property name="text">
      <string>receive</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="lbl_FastPositions">
     <property name="text">
      <string>Fast pos.</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QCheckBox" name="cb_GndFlagReceive">
     <property name="text">
      <string>receive</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="cb_AircraftPartsSend">
     <property name="text">
      <string>send</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QLabel" name="lbl_TextCodec">
     <property name="text">
      <string>Text codec</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QLineEdit" name="le_TextCodec">
     <property name="placeholderText">
      <string>e.g. &quot;latin1&quot;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pb_SetDefaults">
     <property name="text">
      <string>set defaults</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="cb_Override">
     <property name="text">
      <string>override</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lbl_FsdSetup">
     <property name="text">
      <string>FSD setup:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QCheckBox" name="cb_3LetterAirlineICAO">
     <property name="toolTip">
      <string>force 3 letter airline ICAO code</string>
     </property>
     <property name="text">
      <string>3 letter airline</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QCheckBox" name="cb_VisualPositionSend">
     <property name="text">
      <string>Send visual pos.</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QCheckBox" name="cb_EuroscopeSimData">
     <property name="toolTip">
      <string>Read SIMDATA packets from EuroScope simulator session</string>
     </property>
     <property name="text">
      <string>euroscope simdata</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_Override</tabstop>
  <tabstop>pb_SetDefaults</tabstop>
  <tabstop>cb_AircraftPartsSend</tabstop>
  <tabstop>cb_AircraftPartsReceive</tabstop>
  <tabstop>cb_GndFlagSend</tabstop>
  <tabstop>cb_GndFlagReceive</tabstop>
  <tabstop>cb_FastPositionSend</tabstop>
  <tabstop>cb_FastPositionReceive</tabstop>
  <tabstop>cb_VisualPositionSend</tabstop>
  <tabstop>cb_EuroscopeSimData</tabstop>
  <tabstop>le_TextCodec</tabstop>
  <tabstop>cb_3LetterAirlineICAO</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
